subprojects/core/src/test/groovy/org/gradle/internal/fingerprint/impl/EmptyHistoricalFileCollectionFingerprintTest.groovy

```diff
@@ -45,20 +45,20 @@ class EmptyFileCollectionFingerprintTest extends Specification {
             "file2.txt": new DefaultNormalizedFileSnapshot("file2.txt", FileType.RegularFile, HashCode.fromInt(234)),
         ], FingerprintCompareStrategy.ABSOLUTE)
         expect:
-        getChanges(EmptyFileCollectionFingerprint.INSTANCE, fingerprint, false).toList() == [
+        getChanges(EmptyHistoricalFileCollectionFingerprint.INSTANCE, fingerprint, false).toList() == [
             FileChange.removed("file1.txt", "test", FileType.RegularFile),
             FileChange.removed("file2.txt", "test", FileType.RegularFile)
         ]
-        getChanges(EmptyFileCollectionFingerprint.INSTANCE, fingerprint, true).toList() == [
+        getChanges(EmptyHistoricalFileCollectionFingerprint.INSTANCE, fingerprint, true).toList() == [
             FileChange.removed("file1.txt", "test", FileType.RegularFile),
             FileChange.removed("file2.txt", "test", FileType.RegularFile)
         ]
     }
 
     def "comparing to itself works"() {
         expect:
-        getChanges(EmptyFileCollectionFingerprint.INSTANCE, EmptyFileCollectionFingerprint.INSTANCE, false).toList() == []
-        getChanges(EmptyFileCollectionFingerprint.INSTANCE, EmptyFileCollectionFingerprint.INSTANCE, true).toList() == []
+        getChanges(EmptyHistoricalFileCollectionFingerprint.INSTANCE, EmptyHistoricalFileCollectionFingerprint.INSTANCE, false).toList() == []
+        getChanges(EmptyHistoricalFileCollectionFingerprint.INSTANCE, EmptyHistoricalFileCollectionFingerprint.INSTANCE, true).toList() == []
```

Comment:
I guess we would need to compare an `EmptyCurrentFileCollectionFingerprint` to an `EmptyHistoricalFileCollectionFingerprint`, right?